content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md

```diff
@@ -32,7 +32,9 @@ the corresponding `PersistentVolume` is not be deleted. Instead, it is moved to
 
 1. List the PersistentVolumes in your cluster:
 
+       ```
        kubectl get pv
+       ```
```

Comment:
In the [preview](https://deploy-preview-10727--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/change-pv-reclaim-policy/), you can see that there's too much indentation. I think this will render correctly if you indent the command only 4 spaces.

Reply:
The language of the code should be specified as well. In this case it should be `shell`.

```
\`\`\`shell
kubectl get pv
\`\`\`
```